frontend/src/components/Review/ReviewImageUploader/ReviewImageUploader.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Heading, Spacing, Text } from '@fun-eat/design-system';
+import React, { useState } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```

React 전역 말고 타입도 따로 import 하는건 어떤가요?